fms/models/llama.py

```diff
@@ -218,8 +218,6 @@ def __init__(
         if self.config.p_dropout:
             self.dropout = nn.Dropout(self.config.p_dropout)
 
-        self.reset_params()
```

Comment:
it probably does make sense to remove this, but I think the existing training script in this repo expects model instances it gets to be randomly initialized. we could can add a reset_params() call to that code later.

Then again, it's not defined in any super class, so calling code has to hope to find it. PyTorch modules use the convention `reset_parameters` instead of reset_params, should we change to that for the method name, to at least match pytorch?

Reply:
Yeah that's a good idea, I'll do that. The downside is that I can no longer use `hasattr(m, "reset_parameters")` to check the submodules, but arguably I should be listing those submodules out explicitly anyways.

Reply:
what is it that breaks about `hasattr(m, "reset_parameters")`?

I don't currently see any references to `reset_parameters` in fms or `pretrain` repos.

Reply:
related change: https://github.com/foundation-model-stack/foundation-model-stack/pull/179

Reply:
What breaks is that earlier I had been relying on the existence of a `reset_params` method to know when to call it. If every module has a `reset_parameters`, then I can't just call it whenever it occurs, as I could end up custom-initializing at the layer level, then re-initializing at the sublayer level with the wrong init scheme. But again NBD, just means I need to list out the custom modules explicitly in the init scheme